README.md

```diff
@@ -1,20 +1,24 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
+-A click handler executes a function each time event is triggered
 
-(b) Where in your JS file should you put it, and WHY? 
+(b) Where in your JS file should you put it, and WHY?
+-You should put it with the document ready so that it will only work once the all the JS is loaded and the DOM is ready
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+-You haven't referenced the JQuery properly...add the CDN reference in a src tag in the html
 
 ### Question 3
-What should go in the `$(document).ready()` part of your JS file? 
+What should go in the `$(document).ready()` part of your JS file?
+-Everything
```

Comment:
Best practice is to have just your click handlers in your `$(document).ready()` and your function definitions outside. Only the click handlers depend on the HTML being fully loaded.